pyartcd/pyartcd/jenkins.py

```diff
@@ -245,3 +246,17 @@ def start_microshift_sync(version: str, assembly: str, **kwargs):
         },
         **kwargs
     )
+
+
+def start_scan_osh(build_nvrs: str, email: Optional[str] = ""):
```

Comment:
Let's accept **kwargs and pass it along to start_build like we do in above funcs

Reply:
done https://github.com/openshift-eng/aos-cd-jobs/pull/3903/commits/2deab8f5c2506ce582342b6dd578db93ae07cdb1

Reply:
Which use case does this cover/fix?